.storybook/polyfills.js

```diff
@@ -3,3 +3,4 @@ import 'core-js/modules/es6.array.fill';
 import 'core-js/modules/es6.string.includes';
 import 'core-js/modules/es6.string.trim';
 import 'core-js/modules/es7.object.values';
+import './polyfills/element-matches';
```

Comment:
Any way we could do this without using matches? If not, how should we suggest communicating the updated polyfill list here in order to not break teams?

Reply:
@joshblack Two options:

1. Embed the polyfill (Or an equivalent) instead of making it a dependency
2. Use our regular communication channels to advertise the change

Reply:
Took the former approach for now.